lib/RefBackend/RefBackend.cpp

```diff
@@ -22,6 +22,7 @@
 #include "mlir/Dialect/StandardOps/IR/Ops.h"
 #include "mlir/Transforms/DialectConversion.h"
 #include "set"
```

Comment:
nit: (not for this PR): should be `#include <set>` and be near `<numeric>` below.